src/printer.js

```diff
@@ -1204,14 +1204,14 @@ function printClass(path, options, print) {
   const attrs = printAttrs(path, options, print, { inline: isAnonymousClass });
   const declaration = isAnonymousClass ? [] : [...attrs];
 
-  if (node.isReadonly) {
-    declaration.push("readonly ");
-  }
-
   if (node.isFinal) {
     declaration.push("final ");
   }
 
+  if (node.isReadonly) {
+    declaration.push("readonly ");
+  }
+
   if (node.isAbstract) {
```

Comment:
Shouldn't this also be moved up up before `isReadonly`? PER says

> 1. Inheritance modifier: abstract or final
> 2. Visibility modifier: public, protected, or private
> 3. Scope modifier: static
> 4. Mutation modifier: readonly

Reply:
Awesome! let me improve it